query_tests/generate_rnd_queries.py

```diff
@@ -327,6 +335,18 @@ def expr_with_operator(column, provider, inner_type, dimensions):
         return f'{column} IS NULL'
     else:
         op = random.choice(operators)
+        if op in ADDITIONAL_STRING_OPERATORS:
+            if dimensions:
+                # the additional string ops are not supported with arrays, so
+                # use an equality operator with any instead
+                return f'{val} = ANY ("{column}")'
+            else:
+                unquoted_val = provider()
+                compare_to = f"'%{unquoted_val}_'"
```

Comment:
make we could cycle between having the wildcard as prefix and as suffix

Reply:
what would that test additionally?

Reply:
The performance of `%x` and `x%` is very different. Having a wildcard in the beginning is a lot more expensive.

Reply:
Ah so you're saying we should cycle the usage of `%` (instead of `_` as suffix)